.github/workflows/e2e_tests.yml

```diff
@@ -253,13 +253,78 @@ jobs:
           VER: ${{ matrix.version }}
           RESOURCE: ${{ secrets.AZURE_RESOURCE }}
           MINOR_VER: ${{ steps.get_minor_ver.outputs.REL_BRANCH_VER }}
+
+  eksd-e2e-test:
+    name: eks-e2e-test
+    runs-on: ubuntu-latest
+    needs: [create-test-images, eks-e2e-test]
+
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ["1.19", "1.18"]
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: aws-config
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION  }}
+
+      - name: Get the tag
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF/refs\/tags\/test-/}
+        shell: bash
+
+      - name: Get minor version
+        id: get_minor_ver
+        run: echo ::set-output name=REL_BRANCH_VER::$(echo $TAG | grep -Eo '[0-9]+\.[0-9]+')
+        shell: bash
+        env:
+          TAG: ${{ steps.get_tag.outputs.TAG }}
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ steps.get_minor_ver.outputs.REL_BRANCH_VER }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ steps.get_minor_ver.outputs.REL_BRANCH_VER }}-go-
+
+      - name: eksd-cluster-create
+        run: bash ./tetrateci/create_eksd_cluster.sh
+        env:
+          VER: ${{ matrix.version }}
+          MINOR_VER: ${{ steps.get_minor_ver.outputs.REL_BRANCH_VER }}
```

Comment:
Can you bring the new naming used (ISTIO_MINOR_VER and K8S_VERSION) here?

Reply:
Yes, but first I would need to rebase onto the k8s patch, cause I am afraid there would be a bunch of merge conflicts.